meson.build

```diff
@@ -15,6 +17,29 @@ sources = [
   'src/vapoursynth.cpp'
 ]
 
+libs += static_library('p2p_main',
+    [
+        'libp2p/p2p_api.cpp',
+        'libp2p/v210.cpp',
+        'libp2p/simd/cpuinfo_x86.cpp',
+        'libp2p/simd/p2p_simd.cpp'
+    ],
+    include_directories: ['libp2p', 'libp2p/simd'],
+    cpp_args: ['-std=c++14']
```

Comment:
Isn't this already set in the project options?

Reply:
I'm not sure if static_library instance inherits c++ standard definition from entire project, so I set it explicitly. If you sure that was redundant, just make another PR and improve it.